### PR TITLE
Credit line bug

### PR DESCRIPTION
**Pull request by Lucashuang0802**
Fix issue #507. @olleolleolle please give a review. 😃

### CODE REVIEW

**Comment by olleolleolle** [comment]
Thanks! Is it possible to figure out a test for this?

File: lib/github_changelog_generator/generator/generator_generation.rb [hunk 1]
```diff
@@ -21,7 +21,8 @@ def compound_changelog
 
       log += File.read(options[:base]) if File.file?(options[:base])
 
-      log += "\n\n\\* *This Change Log was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*"
+      credit_line = "\* *This Change Log was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*"
+      log += "\n\n\#{credit_line}" if log.include? credit_line == false
```
Comment (Lucashuang0802) [inline lib/github_changelog_generator/generator/generator_generation.rb]: @olleolleolle do you know where the extra '\'? Should it be `"\n\n#{credit_line}"`?

**Comment by Lucashuang0802** [comment]
@olleolleolle perhaps we can check there is only one `credit_line` text the whole file. WDYT?

**Comment by olleolleolle** [comment]
For an integration test approach:

- Generate a file CHANGELOG.md (or have a fixture file) which has 1 credit_line in it
- Re-generate with some data (not important what data, just enough so that the credit_line code will be used)
- Expect that only 1 credit line is in the resulting file

Then, as a follow-up (next) feature:


- Generate a file CHANGELOG.md (or have a fixture file) which has _2 credit_lines_ in it
- Re-generate with some data (not important what data, just enough so that the credit_line code will be used)
- Expect that only 1 credit line is in the resulting file

**Pull request merged**